### PR TITLE
WIP global: alpha release

### PR DESCRIPTION
**Pull request by jacenkow**
* Adds a method to communicate with Beard Celery instances
  in order to delegate a clustering job.

* Adds methods to create a new author profile and update
  signatures to point to new author profiles.

* Adds methods to keep track of new/updated records.

Signed-off-by: Grzegorz Jacenków <grzegorz.jacenkow@cern.ch>

### CODE REVIEW

**Comment by jacenkow** [comment]
Stuff to do

- [ ] Remove `numpy` while calculating the most rich-in-information signature.
- [ ] Update the documentation in `logic.py` and `tasks.py`.
- [ ] Write tests.

However, good enough for a review @kaplun.

File: inspire_disambiguation/logic.py [hunk 1]
```diff
@@ -70,7 +70,7 @@ def _check_if_claimed(signature):
 
 
 def _create_distance_signature(signatures_map, uuid):
-    """Create a signature for distance measurement.
+    """Create a signature for a similarity measurement.
```
Comment (kaplun) [inline inspire_disambiguation/logic.py]: Wasn't distance correct?

Comment (jacenkow) [inline inspire_disambiguation/logic.py]: Distance measurement is a part of `beard-server` and what I wanted to highlight this way, is that we know how are the signatures are grouped together but we leave the `howto` to the *black box*.

**Comment by jacenkow** [comment]
Moved to https://github.com/inspirehep/inspire-next/pull/1353

**Pull request closed without merging**